docs/user/alerting/alerting-setup.asciidoc

```diff
@@ -43,31 +55,52 @@ To access alerting in a space, a user must have access to one of the following f
 * <<xpack-siem,*Security*>>
 * <<uptime-app,*Uptime*>>
 
-See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to these features.
-Also note that a user will need +read+ privileges for the *Actions and Connectors* feature to attach actions to a rule or to edit a rule that has an action attached to it.
+See <<kibana-feature-privileges, feature privileges>> for more information on 
+configuring roles that provide access to these features.
+Also note that a user will need +read+ privileges for the 
+*Actions and Connectors* feature to attach actions to a rule or to edit a rule 
+that has an action attached to it.
 
 [float]
 [[alerting-restricting-actions]]
 ==== Restrict actions
 
-For security reasons you may wish to limit the extent to which {kib} can connect to external services. <<action-settings>> allows you to disable certain <<action-types>> and allowlist the hostnames that {kib} can connect with.
+For security reasons you may wish to limit the extent to which {kib} can connect 
+to external services. <<action-settings>> allows you to disable certain 
+<<action-types>> and allowlist the hostnames that {kib} can connect with.
 
 [float]
 [[alerting-spaces]]
 === Space isolation
 
-Rules and connectors are isolated to the {kib} space in which they were created. A rule or connector created in one space will not be visible in another. 
+Rules and connectors are isolated to the {kib} space in which they were created. 
+A rule or connector created in one space will not be visible in another. 
 
 [float]
 [[alerting-authorization]]
 === Authorization
 
-Rules are authorized using an <<api-keys,API key>> associated with the last user to edit the rule. This API key captures a snapshot of the user's privileges at the time of edit and is subsequently used to run all background tasks associated with the rule, including condition checks like {es} queries and triggered actions. The following rule actions will re-generate the API key:
+Rules are authorized using an <<api-keys,API key>> associated with the last user 
+to edit the rule. This API key captures a snapshot of the user's privileges at 
+the time of the edit. They are subsequently used to run all background tasks associated 
+with the rule, including condition checks like {es} queries and triggered 
+actions. The following rule actions will re-generate the API key:
 
 * Creating a rule
 * Updating a rule
 
+When you disable a rule, it retains the associated API key which is re-used when the 
+rule is enabled. If the API key is missing when you enable the rule, it 
+generates a new key that has your security privileges.
+
+You can update an API key manually in 
+**{stack-manage-app} > {rules-ui}** by selecting **Update API key** in 
+the actions menu of the respective rule.
```

Comment:
You can also update the API key manually from the rule details page. Would that warrant a mention here?

Reply:
Sure! Thanks for the detail, I amended the sentence via [daca621](https://github.com/elastic/kibana/pull/132402/commits/daca6217d8eb38d38c619e538f87074aff51b3fd).